composer/utils/checkpoint.py

```diff
@@ -419,7 +432,7 @@ def read_data(self, plan: LoadPlan, planner: LoadPlanner):
                                                                 storage_reader=storage_reader)
                 state.load_optim_state(optim_state)
 
-        # 3. Optionally, load RNG.
+        # 3. Optionally load RNG
```

Comment:
this is okay to do because there is a tensor in it I guess?

Reply:
yes I think so. Should we move it into initial load?